### PR TITLE
Update splash.html

### PR DESCRIPTION
**Pull request by bluewavenet**
Update splash.html to make it responsive and mobile friendly, and show by example use of Submit button via http form method=get.

### CODE REVIEW

File: resources/splash.html [hunk 1]
```diff
@@ -34,20 +25,72 @@
...
+<span style="color:red; font-style:normal;">
+Welcome!
+</span>
+</b>
+<hr>
+<b><br>For access to the Internet, please click Continue.</b><br><br><hr>
+<form method='get' action='$authaction'>
+<input type='hidden' name='tok' value='$tok'>
+<input type='hidden' name='redir' value='$redir'>
+<input type='submit' value='Continue'></form>
```
Comment (mwarning) [inline resources/splash.html]: better put </form> in another line

File: resources/splash.html [hunk 2]
```diff
@@ -34,20 +25,72 @@
...
+
+	
+	<hr>Copyright (C) 2004-2016.  This software is released under the GNU GPL license.
```
Comment (mwarning) [inline resources/splash.html]: no need for indendation here :-)

File: resources/splash.html [hunk 3]
```diff
@@ -34,20 +25,72 @@
...
+<b>$gatewayname Hotspot Gateway.</b>
+<br><br><b>
+<img src="$imagesdir/splash.jpg" alt="Splash Page:For access to the Internet, please click Continue."> 
+<hr>
+<span style="color:red; font-style:normal;">
+Welcome!
+</span>
+</b>
+<hr>
+<b><br>For access to the Internet, please click Continue.</b><br><br><hr>
```
Comment (mwarning) [inline resources/splash.html]: put the br outside of the b tag.
Also put the hr on it's own lines

File: resources/splash.html [hunk 4]
```diff
@@ -34,20 +25,72 @@
...
+{
+color:black;
+margin-left: 0%;
+margin-right: 5%;
+text-align:left;   
+font-size: 1.0em;
+line-height: 2.5em;
+font-weight: bold;
+border: 1px solid;
+}
```
Comment (mwarning) [inline resources/splash.html]: Can you keep the css shorter?
I noticed you want to make theses examples nice and all. But a uniform browser display is not the important point here.

Comment (bluewavenet) [inline resources/splash.html]: It is not so much uniform browser display, more working in a clean usable way on mobile browsers "out of the box" - and leading by example. Presenting a polished looking user interface will not do the continuing success of the project any harm ;)
ps - I do like your pedantic suggestions for corrections - I'll get onto it :D

Comment (mwarning) [inline resources/splash.html]: I agree with some of your points.

ps - good, I already felt bad about being pedantic xD

**Pull request merged**